nixpkgs_review/utils.py

```diff
@@ -62,3 +63,11 @@ def current_system() -> str:
         text=True,
     )
     return system.stdout
+
+
+def nix_nom_tool() -> str:
```

Comment:
this logic can probably go into the review comment above

> you can resolve nix_flavor into paths to nix and nix-shell here

Reply:
see above